bin/copyRawDataToPrm.sh

```diff
@@ -281,64 +281,72 @@ function splitSamplesheetPerProject() {
 		#  * either only demultiplexing was requested via the samplesheet
 		#  * or when disabled on the commandline by enabling "archiveMode".
 		#
+		log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "mergedSamplesheet = ${mergedSamplesheet}."
 		if [[ "${mergedSamplesheet}" == 'true' ]]
 		then
 			_project=$(echo "${_project}" | grep -Eo 'GS_[0-9]+')
```

Comment:
This means the code now relies on a pattern for the project name to do something "special". Might work for now, but looks brittle...